mmdeploy/codebase/base/task.py

```diff
@@ -302,6 +303,14 @@ def visualize(self,
             show=show_result,
             out_file=output_file)
 
+    def mode(self) -> str:
```

Comment:
could remove this method since we rewrite forward method of single stage and two-stage base class.

Reply:
done.